src/UI/Window.cpp

```diff
@@ -21,23 +21,19 @@
 
 
 
-Byte cWindow::m_WindowIDCounter = 0;
+char cWindow::m_WindowIDCounter = 1;
```

Comment:
`char` overflow is undefined behaviour. Besides, if it's unsigned in all current protocols then why not make it match?